### ISSUE TITLE
Cross origin error in app when clicking hyperlinks

### ISSUE THREAD

**Issue by jrdallen97** [cypress-io/cypress#427]
I'm having an issue where whenever I load a page using a hyperlink Cypress throws an error. This happens both with and without Chrome Web Security enabled, but they throw different errors.

Oddly, if I open a new tab after a test has failed with one of these errors and try to go to my site's URL, Chrome states that the site can't be reached and might be down. The console shows the same error as when I try to click a hyperlink without Chrome Web Security (`net::ERR_CONTENT_DECODING_FAILED`), but doesn't provide the extra details.

Tests still run normally from using `cypress run`, even with video recording enabled, and I get no console errors using the site in Chrome. I can also load pages fine using `.visit()`, so only hyperlinks are causing the error as far as I can tell.

I'm on the newest versions of Cypress (CLI 0.13.1 and App 0.19.0) and tried with an older version (CLI 0.13.0 and App 0.18.5), and am using Chrome 56 (I also tested with 54).

## With Chrome Web Security

Whenever I use `cy.click()` on a hyperlink on our site Cypress throws the error `CypressError: Cypress detected a cross origin error happened on page load:`.

![screen shot 2017-02-13 at 10 34 46](https://cloud.githubusercontent.com/assets/24575899/22881494/a7372fd4-f1de-11e6-9e6d-20e5c34ba060.png)

## Without Chrome Web Security

If I follow the suggestion to disable Chrome Web Security then I get a different error in the same place.

![screen shot 2017-02-13 at 11 20 29](https://cloud.githubusercontent.com/assets/24575899/22881455/7ba4d51a-f1de-11e6-9250-da7c7114bdcf.png)

In the console I also get the following error:

![screen shot 2017-02-13 at 11 22 45](https://cloud.githubusercontent.com/assets/24575899/22881576/1dd1e30a-f1df-11e6-9646-7353c10d7ce0.png)

## Update

This isn't happening any more, but I think it indicates that the app (`cypress open`) behaves differently to the CLI (`cypress run`), because, in theory, I think the two commands should have had the same error. For some reason, running `/Applications/Cypress.app/Contents/MacOS/Cypress --show-headless-gui` worked fine, despite looking the same as `cypress open`.

## Update 2

The issues have resurfaced unfortunately.

**Comment by brian-mann** [comment]
This may have actually been happening due to a bug we fixed in this version:

https://docs.cypress.io/guides/references/changelog.html#0-19-3

Can you check to see if this is still happening? If not we'll close this issue.

**Comment by piotrusin** [comment]
I have the same issue. Using Cypress 0.19.4 and Chrome 60.0.3112.101.

**Comment by Amaal-Mohammed** [comment]
I am having the same issue

**Comment by brian-mann** [comment]
@Amaal-Mohammed is there a way to fix what exactly? The op's original post was about a bug with decoding brotli compressed responses - which we fixed a long, long time ago.

Getting cross origin errors is something that can happen in Cypress if you're trying to load or navigate between super domains (which is an anti pattern).

Our docs explain this here and how to work around it: https://docs.cypress.io/guides/guides/web-security.html#

I'm going to close this issue - as I'm assuming the original bug was fixed long ago.
